inference-engine/src/mkldnn_plugin/nodes/mkldnn_pad_node.cpp

```diff
@@ -227,6 +227,14 @@ void MKLDNNPadNode::createPrimitive() {
     }
 }
 
+bool MKLDNNPadNode::isExecutable() const {
+    return !(isInputTensorAtPortEmpty(0) && isOutputTensorAtPortEmpty(0));
```

Comment:
!isOutputTensorAtPortEmpty(0) seems to be mandatory condition for execution. The condition doesn't guarantee that.

Reply:
agree, fixed